lib/engine/game/g_18_ireland/entities.rb

```diff
@@ -42,6 +42,17 @@ module Entities
             desc: 'The owning Corporation can place a yellow tile without payment of terrain costs.'\
             ' The company closes once this ability has been used twice.',
             sym: 'BoW',
+            abilities: [
+              {
+                type: 'tile_lay',
+                hexes: [],
+                tiles: [],
+                when: 'owning_corp_or_turn',
+                owner_type: 'corporation',
+                free: true,
+                count: 2,
```

Comment:
The text says "The company closes once this ability has been used twice", so I think you want to add `closed_when_used_up` here?

Reply:
yep

Reply:
fixed